src/site/markdown/server/manage-webadmin.md

```diff
@@ -2708,10 +2708,10 @@ To move deleted messages in the vault, you need to specifically configure the De
 
 Here are the following actions available on the 'Deleted Messages Vault'
 
- - [Restore Deleted Messages](#Restore_deleted_messages)
- - [Export Deleted Messages](#Export_deleted_messages)
- - [Purge Deleted Messages](#Purge_deleted_messages)
- - [Permanently remove Deleted Message](#Permanently_remove_deleted_message) 
+ - [Restore Deleted Messages](#restore_deleted_messages)
```

Comment:
I'm not OK changing these links as we rely on github only for convenience and up to date documentation.

The key target is http://james.apache.org/server/manage-webadmin.html which slightly differ from the syntax

Reply:
IMO, it's not friendly for Github users since because nothing happens when clicking those links.

Anyway, great to know that. I will revert them.

Reply:
And as I see on apache doc, markdown is not applied
![Screenshot_2019-05-20_11-26-18](https://user-images.githubusercontent.com/7950991/57996591-901ced80-7af2-11e9-8952-32359c033640.png)

Reply:
I updated pull request